src/components/Tree/Tree.stories.tsx

```diff
@@ -327,3 +328,20 @@ export const WithExpandOnSelect = ({ ...args }: TreeProps) => {
         </div>
     );
 };
+
+export const InsideFlyout = ({ ...args }: TreeProps) => {
+    const [isOpen, setIsOpen] = useState(false);
+
+    return (
+        <div style={{ width: 800 }}>
```

Comment:
Just a detail here but you could use our Layout component instead of div -> either `Box` or `Container`